tests/ceph_volume/test_osd_deploy_with_spec_crush_device_class.py

```diff
@@ -0,0 +1,26 @@
+from cephci.utils.configs import get_configs
+from cli.cephadm.cephadm import CephAdm
+from cli.utilities.operations import wait_for_cluster_health
+from cli.utilities.utils import create_yaml_config
+
+
+def run(ceph_cluster, **kw):
+    """Deploy OSDs using DriveGroup spec crush_device_class"""
+
+    # Get configs
+    get_configs()
+    config = kw.get("config")
+
+    # Get the installer and OSD nodes
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+
+    # Generate a yaml file
+    file = create_yaml_config(node=installer, config=config)
+
+    # Create OSDs with yaml file
+    c = {"pos_args": [], "input": file}
+    CephAdm(nodes=installer, mount=file).ceph.orch.apply(**c)
+
+    # Wait for cluster health to be ok
+    wait_for_cluster_health(node=installer, status="HEALTH_OK")
```

Comment:
Only `HEALTH_OK` validation is sufficient ? Don't need any other validations ?

Reply:
I have added one more validation to check crush_device_class is set correctly. Pls check.